awx/ui/src/screens/WorkflowApproval/WorkflowApprovalList/WorkflowApprovalList.js

```diff
@@ -239,6 +255,41 @@ function WorkflowApprovalsList() {
           />
         </Card>
       </PageSection>
+      {isDeleteModalOpen ? (
+        <AlertModal
+          variant="danger"
+          title={t`Are you sure you want to the following workflow approvals?`}
+          isOpen={isDeleteModalOpen}
+          onClose={() => setIsDeleteModalOpen(false)}
+          actions={[
+            <Button
+              ouiaId="delete-modal-confirm"
+              key="delete"
+              variant="danger"
+              aria-label={t`confirm delete`}
+              onClick={handleDelete}
+            >
+              {t`Delete`}
+            </Button>,
+            <Button
+              ouiaId="delete-cancel"
+              key="cancel"
+              variant="link"
+              aria-label={t`cancel delete`}
+              onClick={() => setIsDeleteModalOpen(false)}
+            >
+              {t`Cancel`}
+            </Button>,
+          ]}
+        >
+          {selected.map((item) => (
+            <span key={item.id} id={`item-to-be-deleted-${item.id}`}>
+              <strong>{item.name}</strong>
```

Comment:
Consider adding a job ID as a way to differentiate which workflow approval will be deleted. It is difficult to discern between them as they may have the same name. 

<img width="1507" alt="image" src="https://user-images.githubusercontent.com/9053044/159355047-bbb5ee51-74c7-4218-b974-e0cdad80520b.png">